modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/TimeUtils.scala

```diff
@@ -0,0 +1,21 @@
+package com.snowplowanalytics.snowplow.enrich.common.fs2
+
+import java.time.Instant
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+import scala.util.Try
+
+object TimeUtils {
```

Comment:
Given the file simplicity, I'd keep these in the BigQueryEncodingUtils file, make the methods package-private.